### PR TITLE
cleanup license, readme and packages

### PR DESCRIPTION
**Pull request by cnishina**
- Remove licenses not in the main directory
- Update the main license to 2018
- Remove versions and repo information from package.json not in the
website directory. reasoning to main this information in one location.
- Improve main readme to point to the book and have information about
the book.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Testing Angular Applications
+Copyright (c) 2018 Testing Angular Applications
```
Comment (jesselpalmer) [inline LICENSE]: What do you think about just changing the copyright date from `2017` to `2017 to present`? That way we won't have to worry about updating it again.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+[Testing Angular Applications](https://www.manning.com/books/testing-angular-applications) is an example-rich, hands-on guide that gives you the real-world techniques you need to thoroughly test all parts of your Angular applications. You'll start with an introduction to Angular, TypeScript, testing fundamentals, and the testing frameworks and tools used in the book and begin writing your first tests. Next, you'll learn how to set-up your Angular environment, including TypeScript and the Angular dependencies and you'll create your first component. The second part of the book focuses on the most important concepts of Angular, such as testing components, directives, pipes, services, and routing. By the end of this book, you'll be able to confidently write unit and end-to-end tests for Angular applications in TypeScript.
```
Comment (jesselpalmer) [inline README.md:13]: Nice! Much better than what is there now.

File: LICENSE (hunk 1 above)
Comment (cnishina) [reply LICENSE]: 2017 to present sounds good.

File: README.md (hunk 2 above)
Comment (cnishina) [reply README.md:13]: 🎉

**Pull request merged**